Implementations/Algorithms/Pong_REINFORCE.py

```diff
@@ -159,9 +217,49 @@ def set_seeds(self, s):
         torch.manual_seed(s)
         random.seed(s)
 
+class DiffFrame(gym.ObservationWrapper):
+    def __init__(self, env):
+        super().__init__(env)
+
+    @staticmethod
+    def prepro(I):
```

Comment:
is this method the same as above? why are they repeated? Shouldn't it be better to organize them on a helper module and keep only one implementation?